CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 ## How to Contribute to SecureDrop Docs
```

Comment:
Maybe this should explicitly say "SecureDrop Developer Docs"?

Reply:
Done.